### PR TITLE
Examples/certificate from file

### PR DESCRIPTION
**Pull request by vidavidorra**
#### Description
Add example for verification, which also reads certificates from local files. Also organise the examples a bit.

### CODE REVIEW

**Comment by daenney** [comment]
```
examples/util/util.go:81: G304: Potential file inclusion via variable (gosec)

	rawData, err := ioutil.ReadFile(path)

examples/util/util.go:111: G304: Potential file inclusion via variable (gosec)

	rawData, err := ioutil.ReadFile(path)

examples/util/util.go:88:25: ST1005: error strings should not be capitalized (stylecheck)

		return nil, errors.New("Block is not a private key, unable to load key")

		                      ^

examples/util/util.go:99:26: ST1005: error strings should not be capitalized (stylecheck)

			return nil, errors.New("Unknown key time in PKCS#8 wrapping, unable to load key")

			                      ^

examples/util/util.go:106:24: ST1005: error strings should not be capitalized (stylecheck)

	return nil, errors.New("No private key found, unable to load key")

	                      ^

The command "golangci-lint run ./..." exited with 1.
```

**Comment by vidavidorra** [comment]
@daenney Yeah, I saw. I've now setup VSCode with the same linter (golangci-lint) so I can see these locally :)
I've fixed the errors in the latest commit.

**Comment by vidavidorra** [comment]
@at-wat Good catch! Updated the README.

File: examples/certificates/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+$ openssl x509 -req -in "${CLIENT_NAME}.csr" -extfile "${EXTFILE}" -days 365 -CA "${SERVER_NAME}.pub.pem" -CAkey "${SERVER_NAME}.pem" -set_serial '0xabcd' -sha256 -out "${CLIENT_NAME}.pub.pem"
+
+# Cleanup.
+$ rm "${EXTFILE}" "${SERVER_NAME}.csr" "${CLIENT_NAME}.csr"
+```
+
+
+EXTFILE='extfile.conf' && echo 'subjectAltName = IP:127.0.0.1\nbasicConstraints = critical,CA:true' > "${EXTFILE}" && SERVER_NAME='server' && openssl ecparam -name prime256v1 -genkey -noout -out "${SERVER_NAME}.pem" && openssl req -key "${SERVER_NAME}.pem" -new -sha256 -subj '/C=NL' -out "${SERVER_NAME}.csr" && openssl x509 -req -in "${SERVER_NAME}.csr" -extfile "${EXTFILE}" -days 365 -signkey "${SERVER_NAME}.pem" -sha256 -out "${SERVER_NAME}.pub.pem" && CLIENT_NAME='client' && openssl ecparam -name prime256v1 -genkey -noout -out "${CLIENT_NAME}.pem" && openssl req -key "${CLIENT_NAME}.pem" -new -sha256 -subj '/C=NL' -out "${CLIENT_NAME}.csr" && openssl x509 -req -in "${CLIENT_NAME}.csr" -extfile "${EXTFILE}" -days 365 -CA "${SERVER_NAME}.pub.pem" -CAkey "${SERVER_NAME}.pem" -set_serial '0xabcd' -sha256 -out "${CLIENT_NAME}.pub.pem"
```
Comment (at-wat) [inline examples/certificates/README.md]: This part seems be a duplicate of above code blocks.

Comment (vidavidorra) [reply examples/certificates/README.md]: Yeah, will fix! I need to stop putting test code like this in the tracked files of a repo ;)

This indeed is a combination of all above commands. I was working on other projects this weekend, but will fix this today together with squashing the commits. I've never done that, but found https://www.internalpointers.com/post/squash-commits-into-one-git so will give it a go. Will be a good learning experience of a git feature I've never used.

As for the commit messages, I write those from the command line with the `-m` option and due to the restriction of the checks (see Slack discussion on improving that with Conventional Commtis) I need to write multiple lines. Using the `-m` option multiple times adds these blank lines.

Comment (jdbruijn) [reply examples/certificates/README.md]: Okay, now this should be fixed. I've done the changed and squashed some of the commits.

**Comment by jdbruijn** [comment]
CI doesn't seem to be triggering properly: https://travis-ci.org/github/pion/dtls/pull_requests

**Comment by at-wat** [comment]
> CI doesn't seem to be triggering properly:

It's due to conflicts with master.
Do you mind rebasing latest master?

**Pull request merged**